packages/block-editor/src/components/inserter/block-patterns-tab/pattern-category-previews.js

```diff
@@ -130,13 +132,25 @@ export function PatternCategoryPreviews( {
 
 	return (
 		<>
-			<VStack
-				spacing={ 2 }
-				className="block-editor-inserter__patterns-category-panel-header"
-			>
+			<VStack className="block-editor-inserter__patterns-category-panel-header">
 				<HStack>
+					<NavigatorBackButton
+						style={
+							// TODO: This style override is also used in ToolsPanelHeader.
+							// It should be supported out-of-the-box by Button.
+							{ minWidth: 24, padding: 0 }
+						}
```

Comment:
Looks like it is supported out of the box now. If we remove these lines the back button looks the same. Or maybe it was only in the context of the ToolsPanelHeader that it was an issue.